src/server/verification/verificationAPI.js

```diff
@@ -112,10 +112,13 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
             }
           }
         } else {
-          //only approved users can do the process
-          const isApprovedToClaim = user.claimQueue && ['approved', 'whitelisted'].includes(user.claimQueue.status)
-          if (conf.claimQueueAllowed > 0 && isApprovedToClaim === false)
+          const isApprovedToClaim = ['approved', 'whitelisted'].includes(get(user, 'claimQueue.status'))
```

Comment:
just prettified a bit this code block

Reply:
ok. i actually prefer the === notation and not using '!'

Reply:
the near code lines are long too so yep, we could use === notation